File: Android/Tejash11_TejashSeth_2024csit1065_2/FunctionAndArrays/AnyBaseToDecimal.java
import java.util.*;

public class Main {

  public static void main(String[] args) {
    Scanner scn = new Scanner(System.in);
    int n = scn.nextInt();
    int b = scn.nextInt();
    int dn = getValueInBase(n, b);
    System.out.println(dn);
  }

  public static int getValueInBase(int n, int b) {
    int i=0,sum=0;
    while (n > 0)
    {
        sum += (n%10)*Math.pow(b,i);
        i++;
        n = n / 10;
    }
    return sum;
  }
}

File: Web/Akanksha0401_AkankshaMishra_2024cs1126_2/FunctionsAndArrays/anybasetodecimal.cpp
#include<iostream>
#include<math.h>
using namespace std;

int main() {
  int n;
  int b;
  cin >> n;
  cin >> b; 
  int i=0,digit,sum=0;
  while(n!=0)
  {
      digit=n%10;
      sum+=pow(b,i)*digit;
      n=n/10;
      i++;
  }
 
  cout <<sum  << endl;
}